channels/1ds-post-js/src/HttpManager.ts

```diff
@@ -222,6 +223,15 @@ export class HttpManager {
                         }
     
                         _xhrTimeout = channelConfig.xhrTimeout;
+                        
+                        const csStream = getInst("CompressionStream");
+
+                        // Controls whether payload compression (gzip) is enabled.
+                        _zipPayload = isFeatureEnabled("zipPayload", coreConfig);
+                        if (!isFunction(csStream) || _sendHook) {
```

Comment:
Lets add an additional comment for the team to note what the `_sendHook` is and why we are disabling if there is a payload processor. 
i.e. as the payload processor may have been added to provide application level compression, we don't want to cause the payload to get corrupted.